rllib/BUILD

```diff
@@ -1354,6 +1354,13 @@ py_test(
     srcs = ["tests/test_dependency_torch.py"]
 )
 
+py_test(
+    name = "tests/test_nn_framework_import_errors",
+    tags = ["team:ml", "tests_dir", "tests_dir_D"],
+    size = "small",
+    srcs = ["teststest_nn_framework_import_errors.py"]
```

Comment:
1. the directory is wrong right? cc @sven1977 
2. there seems to be a typo in srcs?